### DISCUSSION TITLE
Change sender of PreTalx emails to another email address (or one without noreply@)

### DISCUSSION THREAD

**Discussion by marcwalsh** [pretalx/pretalx#1251] [Ideas] [answered]
I've seen a couple of support emails come through to our general mailbox saying they were unsure if they could respond to emails as the email it's sent from is noreply@pretalx.com

I wonder what the best way would be to resolve this considering that the event setup asks for a default email address? 

Would it be possible to change the pretalx.com email to something more inviting to responses (maybe only if a default reply-to is set up) or would it be possible to set the email address that MailJet uses?

**Comment by rixx** [comment, marked as answer]
I'm afraid changing the default sender address on pretalx.com won't be possible – it's not feasible for us to serve as address for all speakers of all events, and in our experience, people tend to ignore the Reply-To address (which we always send!) when the sender address does not include a noreply address.

The current way for an event to use a more welcoming email address would be to use their own email server, and thus their own sender address, which would remove the need for a specific Reply-To address.

**Comment by marcwalsh** [comment]
Thanks @rixx - that makes a lot of sense, and maybe something for us to explore next year!
